cirq-google/cirq_google/api/v2/sweeps.py

```diff
@@ -63,7 +91,10 @@ def sweep_to_proto(
             out.single_sweep.parameter.units = sweep.metadata.units
     elif isinstance(sweep, cirq.Points) and not isinstance(sweep.key, sympy.Expr):
         out.single_sweep.parameter_key = sweep.key
-        out.single_sweep.points.points.extend(sweep.points)
+        if len(sweep.points) == 1 and sweep.points[0] is None:
```

Comment:
Should this support arbitrary const values, not just `None`? If so, can you add a test for the new logic

Reply:
Right, it can. But I found other places used sweep with single point as well. So I add this extra condition to avoid the change of previous behavior. (Add the comment in the code base). I think we can remove this `is None` later when it becomes necessary?

Reply:
How many places in cirq use const sweep points? If they're only tests, can we change them?

Reply:
In cirq_server we always use `sweep_from_proto` to decode from `sweep_to_proto` so this should be fine.